doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -0,0 +1,169 @@
+Upgrade to Flower Next
+======================
+
+Welcome to the migration guide for updating Flower to Flower Next! Whether you're a seasoned user
+or just getting started, this guide will help you smoothly transition your existing setup to take
+advantage of the latest features and improvements in Flower Next, starting from version 1.8. Let's dive in!
+
```

Comment:
Should we mention here at the beginning that this guides is about the "compatibility layer" in Flower Next, and that another guide will follow (or this will be updated) showing how to run Flower Next end-to-end ?

Reply:
Yes, good point. I've added it.